parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -413,6 +413,11 @@ try
         Write-Log "Disable Internet Explorer compat mode and set homepage"
         Set-Explorer
 
+        # if multple LB policies are included for same endpoint then HNS hangs.
+        # this fix forces an error  
+        Write-Host "Enable a HNS fix in 2021-2C+"

```

Comment:
@jsturtevant do you mean that the code in configure-windows-vhd.ps1 does not work?

Reply:
vhd should work, but we also have cases where VHD is not required in aks-engine so wanted to make sure it was enabled for those